src/finance-page/finance-page.css

```diff
@@ -1,19 +1,10 @@
-.full-width {
-    margin: 0 auto;
-    width: 100%;
-}
-
 .finance-page {
-    position: relative;
-}
-
-.page-content {
-    position: fixed;
-    top: 8em;
-
-    padding-bottom: 4em;
-}
-.row-container {
+    height: 100%;
+    min-height: 100vh;
```

Comment:
please use vh only on top level elements.

Reply:
But technically finance-page is the top level, as App component has only font styles, and .root component has no styles as all.